docs/api/doc-block-primary.md

```diff
@@ -29,6 +29,12 @@ import { Primary } from '@storybook/blocks';
 
 `Primary` is configured with the following props:
 
+### `of`
+
+Type: CSF file exports
+
+Specifies the primary (first) story to be rendered.
```

Comment:
Can we state something similar to [Meta's `of` prop description](https://storybook.js.org/docs/react/api/doc-block-meta#of)?

```suggestion
Specifies which CSF file is used to find the first story, which is then rendered by this block. Pass the full set of exports from the CSF file (not the default export!).
```

Reply:
Yeah sure thing, I'll push a commit with the other fix today.